fastapi/.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+max-line-length = 89
+exclude=.venv,.git
+ignore = W503
+extend-ignore =
+    # See https://github.com/PyCQA/pycodestyle/issues/373
+    E203,
```

Comment:
would you be interesting in migrating the linters to use Ruff? :D

Reply:
oc! let me update it